src/site/_includes/course.njk

```diff
@@ -27,13 +27,9 @@ CSS_ORIGIN: 'next'
   Grab the ToC contents for headings ToC. This powers the list of headings
   that appears on the right-hand side of the page. Unfortunately our use of
   the word 'toc' is a bit overloaded :P
-  TODO: The courseToc filter is only used to define the ToC for courses, but
-  it's basically the same thing we use on developer.chrome.com and it would be
-  nice to use it for articles as well. Currently we have a different toc design
-  for those but it's a bit clunky and we should replace it with this.
 #}
 {% set tocContents %}
-  {{- content | courseToc | safe -}}
+  {{- content | toc | safe -}}
```

Comment:
The All Patterns title is misleading here, as it does not list All patterns as in breadcrumb, bot only 1 suide of patterns. Also, "All" is repeated. (it should be .e.g. Layout Patterns, on nothing at all):

![image](https://user-images.githubusercontent.com/1914261/150783646-f83900d2-9e47-4503-b512-8f54963b43f7.png)